lib/keyStorage.js

```diff
@@ -0,0 +1,132 @@
+/*
+ * Key storage for Continuity2017 consensus method.
+ *
+ * Copyright (c) 2017 Digital Bazaar, Inc. All rights reserved.
+ */
+const async = require('async');
+const bedrock = require('bedrock');
+const database = require('bedrock-mongodb');
+const logger = require('./logger');
+const BedrockError = bedrock.util.BedrockError;
+
+// module API
+const api = {};
+module.exports = api;
+
+bedrock.events.on('bedrock-mongodb.ready', callback => async.auto({
+  openCollections: callback => database.openCollections(
+    ['continuity2017_key'], callback),
+  createIndexes: ['openCollections', (results, callback) =>
+    database.createIndexes([{
+      collection: 'continuity2017_key',
+      fields: {id: 1, ledgerNodeId: 1},
```

Comment:
Swap `id` and `ledgerNodeId` order. Allows queries on keys known by a particular ledger node whereas the other order doesn't seem that useful.